_episodes/40.md

```diff
@@ -0,0 +1,495 @@
+interoperable read support. A typical usage was Trino alongside other query 
+engines like Apache Spark which supported many of the data modification language
+(DML) SQL features on Iceberg. One of the biggest requests we got as adoption
+increased was the ability to do everything through just Trino. This episode
+dives into some of the latest features that were missing from the early
+iterations of the Iceberg connector, and what has changed in Iceberg as well!
+
+### What is Apache Iceberg?
+
+[Iceberg](https://iceberg.apache.org/) is a next-generation table format that 
+defines a standard around the metadata used to map data to a SQL query engine. 
+It addresses a lot of the maintainability and reliability issues many engineers
+experienced with the way
+[Hive modeled SQL tables]({% post_url 2020-10-20-intro-to-hive-connector %}) 
+over big data files. 
+
+One common confusion to point out is that table format is not equivalent to file
+formats like ORC or Parquet. The table format is the layer that maintains 
+metadata mapping these files to the concept of a table and other common database
+abstractions.
+
+This episode assumes you hsome basic knowledge of Trino and Iceberg already. If
+you are new to Iceberg or need a refresher, we recommend the two older episodes
+about Iceberg and Trino basics:
+
+* [14: Iceberg: March of the Trinos]({{site.url}}/episodes/14.html)
+* [15: Iceberg right ahead!]({{site.url}}/episodes/15.html)
+
+### Why Iceberg over other formats?
+
+Iceberg is one of three of the recent table formats. There is also
+[Delta Lake](https://delta.io) and [Apache Hudi](https://hudi.apache.org), which
+Trino also supports. DeltaLake recently had 
+[a lot of new features open sourced](https://www.databricks.com/blog/2022/06/30/open-sourcing-all-of-delta-lake.html)
+to catch up with Iceberg. While this was a great advancement for the Delta Lake
+community, there are still some missing capabilities that Delta Lake and Hudi 
+have as they remain Hive-like formats by designing the systems to remain 
+backwards compatible actually reintroduces some limitations that existed in
+Hive. 
+
+On top of that, Iceberg has the most support for query engines and puts a heavy 
+emphasis on being a format that is interoperable. This improves the level of
+flexibility users have to address a wider array of use cases that may involve
+querying over a data warehouse like Snowflake or a data lake running with
+Iceberg. All of this is made possibly by the
+[Iceberg specification](https://iceberg.apache.org/spec) that all these query
+engines must follow.
+
+Finally, a great video presented by Ryan Blue that dives into Iceberg is, 
+"[Why you shouldn't care about Iceberg](https://www.youtube.com/watch?v=_GW3GYZK66U)."
+
+### Metadata Catalogs
+
+Catalogs, in the context of Iceberg, refers to the central storage of the
+metadata and providing the atomic compare-and-swap need to support 
```

Comment:
Start a new sentence after metadata and reword .. I am not sure what you are trying to say